Toshi/Source/Toshi/File/TFile.h

```diff
@@ -1,14 +1,16 @@
 #pragma once
 #include "Toshi/Utils/TSingleton.h"
 #include "Toshi/Strings/TCString.h"
+#include "Toshi/Core/TGenericDList.h"
 
 #ifdef CreateFile
 #undef CreateFile
 #endif
-#include <Toshi/Core/TGenericDList.h>
```

Comment:
triangular quotes are usually used when a header file is from different project

Reply:
Visual Studio Alt+Enter's fault lol. I'm lazy even though i knew it was wrong